### PR TITLE
Feature -1 is completed

### PR DESCRIPTION
**Pull request by Lavanya1991827**
feature-1 development is completed
              ->added user sign button
	     ->added login functionality

### CODE REVIEW

File: README.MD [hunk 1]
```diff
@@ -1,2 +1,7 @@
 GIT
-THis is the first commit from main branch
\ No newline at end of file
+THis is the first commit from main branch
+
+#feature-1
+developer is doing some development  #commit-1 from feature-1 branch
```
Comment (Lavanyact002) [inline README.MD:5]: is it verified?

**Review by Lavanyact002 — COMMENTED** [review]
please check comments and do modifications accordingly.

File: README.MD (hunk 1 above)
Comment (Lavanya1991827) [reply README.MD:5]: verified....

**Review by Lavanyact002 — APPROVED** [review]

**Review by Lavanya1991827 — COMMENTED** [review]
pls check

**Review by Lavanyact002 — APPROVED** [review]
approved...

**Review by Lavanyact002 — APPROVED** [review]
approved...

**Pull request merged**